src/libs/ReportUtils.js

```diff
@@ -1418,9 +1418,10 @@ function getTransactionReportName(reportAction) {
  *
  * @param {Object} report
  * @param {Object} [reportAction={}]
+ * @param {Boolean} [shouldConsiderReceiptBeingScanned=false]
  * @returns  {String}
  */
-function getReportPreviewMessage(report, reportAction = {}) {
+function getReportPreviewMessage(report, reportAction = {}, shouldConsiderReceiptBeingScanned = false) {
```

Comment:
When would `shouldConsiderReceiptBeingScanned` be false?

Reply:
https://github.com/Expensify/App/pull/25906#discussion_r1315720783

Reply:
Ah I see. Thanks for the link